core/classes/Queue/Queue.php

```diff
@@ -96,6 +96,9 @@ public static function process(): array {
                 $fragment = '';
 
                 if (class_exists($task['task'])) {
+                    DB::getInstance()->query('START TRANSACTION');
```

Comment:
I'm not sure if this will work, if PDO remains in autocommit mode? I think you should use `PDO::beginTransaction` instead

Reply:
I did some manual testing and it seemed fine, but I'd agree doing this via PDO would be nicer

Reply:
Done 👍